.github/workflows/release_conductor.yml

```diff
@@ -0,0 +1,41 @@
+name: Assign Release Conductor
+
+on:
+  pull_request:
+
+jobs:
+  pagerduty:
+    if: github.event_name == 'pull_request' && github.event.pull_request.head.ref == 'changeset-release/main'
+    name: Lookup who is oncall from pagerduty
+    uses: primer/.github/.github/workflows/pagerduty_oncall.yml@v2.2.0
```

Comment:
Dependent on https://github.com/primer/.github/pull/48 merging and setting up a new release